tests/tensorflow/quantization/test_algorithm_quantization.py

```diff
@@ -11,23 +11,25 @@
  limitations under the License.
 """
 
+import pytest
 import tensorflow as tf
 from tensorflow.python.keras import layers
-import pytest
 
 from nncf.tensorflow.graph.metatypes.matcher import get_keras_layer_metatype
 from nncf.tensorflow.layers.custom_objects import NNCF_QUANTIZATION_OPERATONS
 from nncf.tensorflow.layers.operation import InputType
 from nncf.tensorflow.layers.wrapper import NNCFWrapper
 from nncf.tensorflow.quantization import FakeQuantize
 from nncf.tensorflow.quantization.algorithm import QuantizationController
-from nncf.tensorflow.quantization.quantizers import Quantizer, TFQuantizerSpec
+from nncf.tensorflow.quantization.quantizers import Quantizer
+from nncf.tensorflow.quantization.quantizers import TFQuantizerSpec
 from tests.tensorflow.helpers import create_compressed_model_and_algo_for_test
 from tests.tensorflow.helpers import get_basic_conv_test_model
 from tests.tensorflow.quantization.utils import get_basic_quantization_config
+# TODO(nlyalyus): WA for the bug 58886, QuantizationMode should be imported after nncf.tensorflow.
+#  Otherwise test_quantize_inputs and test_quantize_outputs_removal will fail, because of invalid inputs quantization
 from nncf.common.quantization.structs import QuantizationMode
```

Comment:
A sensitive import from common.
IDE played a bad joke with me, It moved this import before tensorflow ones and some tests with input quantization started to fail.
Created a ticket for this issue - 58886